components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sync/WorkManagerSyncManager.kt

```diff
@@ -135,7 +135,7 @@ class WorkManagerSyncDispatcher(
         return isSyncActive
     }
 
-    override fun syncNow(startup: Boolean) {
+    override fun syncNow(startup: Boolean, debounce: Boolean) {
         logger.debug("Immediate sync requested, startup = $startup")
         val delayMs = if (startup) {
             // Startup delay is there to avoid SQLITE_BUSY crashes, since we currently do a poor job
```

Comment:
Unrelated nit: we can remove this FIXME.

Reply:
Heh, I hope so!